src/Commands/SyncCommand.php

```diff
@@ -28,7 +28,7 @@ public function handle()
 
         $this
             ->syncScheduledTasksWithDatabase()
```

Comment:
Isn't it strange that the new option is supported for Oh Dear, but not for the local db? I think it would be better if passing the new option would have the same effect for the local db and Oh Dear. 

Let's also keep the names of the methods the same. Maybe `storeScheduledTasksInDatabase`, `storeMonitoredScheduledTasksInOhDear`?